docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -2,34 +2,18 @@
 
 = Adopting Networker services to the {rhos_acro} data plane
 
-If you have `Networker` nodes in your existing {rhos_acro} deployment, you
-should adopt them to the {rhos_acro} data plane too. The procedure is largely
-the same as for `Compute` nodes.
+If you have Networker nodes in your existing {rhos_long} deployment, adopt them to the {rhos_acro} data plane. You decide which services you want to run on the Networker nodes, and then create a separate `OpenStackDataPlaneNodeSet` custom resource (CR) just for these nodes. You might also decide to implement the following options if they apply to your environment:
 
-The main difference between `Networker` and `Compute` nodes is that `Networker`
-nodes do not run `nova-compute` and `libvirt` services.
+* Depending on your topology, you might need to run the `neutron-metadata` service on the nodes, specifically when you want to serve metadata to SR-IOV ports that are hosted on Compute nodes.
 
-You won't need to run `neutron-sriov` service on `Networker` nodes either. But
-depending on your topology, you might need to run `neutron-metadata` service
-there. Specifically, when you'd like to serve metadata to SR-IOV ports hosted
-on `Compute` nodes.
+* If you want to continue running OVN gateway services on Networker nodes, keep `ovn` service in the list to deploy.
 
-Assuming you'd like to continue running OVN gateway services on `Networker`
-nodes, you will keep `ovn` service in the list to deploy. (You may later decide
-to deprovision OVN gateway services from `Networker` nodes and run them on OCP
-nodes instead. This is currently not recommended and is out of scope for the
-adoption procedure.)
+* Optional: You can run the `neutron-dhcp` service on your Networker nodes instead of your Compute nodes. Using `neutron-dhcp` with OVN is not generally needed. However, it might be necessary in deployments with dynamic host configuration protocol (DHCP) relays, or if you require advanced DHCP options that are supported by dnsmasq but not by the OVN native DHCP implementation.
```

Comment:
"Using `neutron-dhcp` with OVN is not generally needed." is passive.
Maybe:
"You might not need to use `neutron-dhcp` with OVN, unless your deployment uses DHCP relays, or advanced DHCP options that are supported by dnsmasq but not by the OVN DHCP implementation."

I wouldn't bother ever spelling out "dynamic host configuration protocol" for DHCP